pkgs/development/python-modules/django-q2/default.nix

```diff
@@ -81,6 +81,37 @@ buildPythonPackage rec {
   disabledTests = [
     # requires a running mongodb
     "test_mongo"
+    # flaky / timing sensitive
+    "test_max_rss"
+    "test_recycle"
+    "test_timeout"
+    # requires network access
```

Comment:
You are sure this is not just local network access aka connecting most likely to itself? Last time I tested this it worked fine on Linux

Reply:
This pops up during these specific tests:
```sh
UserWarning: Retry and timeout are misconfigured. Set retry larger than timeout,failure to do so will cause the tasks to be retriggered before completion.See https://django-q2.readthedocs.io/en/master/configure.html#retry for details.
```

And these three test failures are all cases of:
```sh
>       assert s.reincarnations == 1
E       assert 0 == 1
E        +  where 0 = <django_q.cluster.Sentinel object at 0x10558ee50>.reincarnations
```

The pattern of failures varies depending on the host machine's speed, thus flaky.

(And yes, I tested several times.)

Reply:
If you would prefer, I can gate these three on hostPlatform.isDarwin

Reply:
I mean did you try setting __darwinAllowLocalNetworking to true? 

Also yes, we definitely want them to be Darwin specific

Reply:
Allow local networking: Done
Darwin-specific: Done

Reply:
The local networking for darwin should fix some of the tests. I retested this locally and pushed more minimal version.